proto/gk/v1/roles.proto

```diff
@@ -41,6 +48,11 @@ message Role {
 	repeated Permission permissions = 3;
 }
 
+message RolesAddPermissionRequest {
+	string role_id = 1;
+	string id      = 2;
```

Comment:
Can we name this `permission_id` please? Unless it's very clear it's better to avoid ambiguous `id` fields.